packages/perseus/src/widgets/dropdown/score-dropdown.test.ts

```diff
@@ -7,22 +7,6 @@ import type {
 } from "../../validation.types";
 
 describe("scoreDropdown", () => {
-    it("returns invalid for user input of 0", () => {
-        // Arrange
-        const userInput: PerseusDropdownUserInput = {
-            value: 0,
-        };
-        const rubric: PerseusDropdownRubric = {
-            choices: question1.widgets["dropdown 1"].options.choices,
-        };
-
-        // Act
-        const result = scoreDropdown(userInput, rubric);
-
-        // Assert
-        expect(result).toHaveInvalidInput();
-    });
```

Comment:
Didn't keep a check for this in scoring test since it looks like we might be doing validation before scoring (love that)